src/objective-c/GRPCClient/GRPCCallOptions.h

```diff
@@ -314,6 +326,24 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property(readwrite) BOOL retryEnabled;
 
+/**
+ * Maximum interval in seconds between two consecutive retries. Pass 0 to use default.
+ * Internal-only property used for GTMSessionFetcher transport retry policy.
+ */
+@property(readwrite) NSTimeInterval maxRetryInterval;
+
+/**
+ * Minimum interval in seconds between two consecutive retries. Pass 0 to use default.
+ * Internal-only property used for GTMSessionFetcher transport retry policy.
+ */
+@property(readwrite) NSTimeInterval minRetryInterval;
+
+/**
+ * Multiplier used to increase the interval between retries. Pass 0 to use default.
+ * Internal-only property used for GTMSessionFetcher transport retry policy.
+ */
+@property(readwrite) double retryFactor;
```

Comment:
These properties are implicitly declared `atomic` but their implementations are not atomic. I recommend declaring these properties `nonatomic` (in both the mutable and immutable interfaces).

Reply:
https://github.com/grpc/grpc-ios/issues/50